ignite3/src/main/java/site/ycsb/db/ignite3/IgniteStreamerClient.java

```diff
@@ -58,20 +51,12 @@ public class IgniteStreamerClient extends IgniteAbstractClient {
   public void init() throws DBException {
     super.init();
 
-    INIT_COUNT.incrementAndGet();
-
-    synchronized (IgniteStreamerClient.class) {
-      if (rvPublisher != null) {
-        return;
-      }
```

Comment:
No need to synchronize since `rvPublisher` and `rvStreamerFut` are not static objects. This logic remained unchanged since we updated these variable to be non-static.